DOC: error into how-to-write-unit-tests.md

<!--

IMPORTANT NOTE
--------------

Please note that this entry field uses [Markdown][1], which means that you
need to escape any code snippets pasted here using fenced code blocks,
i.e. by putting ``` before and after it. Also escape, using
backslash, i.e. `\<` any angle brackets to prevent them from being
interpreted as HTML tags.

Use the Preview tab to review your issue before submitting it to verify that
your formatting is correct.

[1]: https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

---

Describe the bug:
A clear and concise description of what the bug is.

Expected vs observed behaviour:
Please describe what you expected to happen and what actually happens. Make
sure to explain what exactly is the problem, just attaching a screenshot is
not always sufficient. When in doubt, please provide more details rather
than too few.

Patch or snippet allowing to reproduce the problem:
Please attach the smallest possible patch allowing to reproduce the problem
in a sample. If there is absolutely no appropriate sample, please attach a
minimal self-contained example in a single file.

Skip this step if the problem can be reproduced in one of the samples
without any changes.

To Reproduce:
Steps to reproduce the behaviour, please make them as detailed as possible.
For example:

1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error
-->

### Description
<!-- Describe the bug here -->
I looked into here:
https://github.com/wxWidgets/wxWidgets/blob/master/docs/contributing/how-to-write-unit-tests.md
for getting the information for building the tests under CMake.
As it is described, I added option `-DwxBUILD_TESTS=ON` to the command line, but I'm getting this error message:

```
CMake Error at build/cmake/functions.cmake:944 (message):
  Invalid value "ON" for option wxBUILD_TESTS.  Valid values are:
  CONSOLE_ONLY ALL OFF
Call Stack (most recent call first):
  build/cmake/options.cmake:15 (wx_option)
  build/cmake/main.cmake:16 (include)
  CMakeLists.txt:103 (include)
```

So, it seems that the documentation file just needs to be updated by changing:

`-DwxBUILD_TESTS=ON` -> `-DwxBUILD_TESTS=ALL`
`-DwxBUILD_TESTS=CONSOLE` -> `-DwxBUILD_TESTS=CONSOLE_ONLY`

Or the CMake script could be fixed for using the switches described into the documentation.
Both solutions would be good.

### Platform and version information

- wxWidgets version <!-- [e.g. 3.2.1] --> you use: git e8fdb66 
- wxWidgets port <!-- [e.g. wxMSW, wxGTK, wxOSX] --> you use: wxGTK
- OS <!-- [e.g. Windows 10, Ubuntu 22.10, macOS 15] --> and its version: Windows

--- a/docs/contributing/how-to-write-unit-tests.md
+++ b/docs/contributing/how-to-write-unit-tests.md
@@ -27,7 +27,7 @@ build the library:
   directory where you ran configure, not the one in the source tree) and run
   `make test` (non-GUI tests) or `make test_gui` (GUI ones) to build.
 
-- When using CMake, add `-DwxBUILD_TESTS=ON` (or `=CONSOLE` for non-GUI tests
+- When using CMake, add `-DwxBUILD_TESTS=ALL` (or `=CONSOLE_ONLY` for non-GUI tests
   only) to the command line arguments, or choose the desired `wxBUILD_TESTS`
   option in `cmake-gui`.
 

